src/internal/starlark/util.go

```diff
@@ -0,0 +1,13 @@
+package starlark
+
+import "go.starlark.net/starlark"
+
+func Union(dicts ...starlark.StringDict) starlark.StringDict {
+	result := make(starlark.StringDict)
+	for _, d := range dicts {
+		for k, v := range d {
+			result[k] = v
```

Comment:
Is there any concern here that union is not commutative?

Reply:
I don't think you can combine dicts in a commutative way.  This is used later on to add global modules that are predefined in each execution environment.